doc/ref_arch/kubernetes/chapters/chapter06.md

```diff
@@ -16,7 +16,9 @@ The following functionalities are recommended for securing kubernetes platform;
 - Image Signing
 - Role-base Access Control
 - Secret Managment
-- Platform multitenant security
+- Kubernetes Cluster Multi-tenancy security
+  - Tenant can be distinct teams or workload types (Develpoment or Production) within an organisation, each with a namespace
+  - Tenant can also be per dedicated kubernetes cluster
```

Comment:
@manamsamuel Are you suggesting a kubernetes cluster dedicated to a tenant?

Reply:
This is also a possible use case depending on the workload type.

Reply:
@manamsamuel Can you address? Thanks